platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapView.java

```diff
@@ -521,13 +521,15 @@ public void onSurfaceTextureUpdated(SurfaceTexture surface) {
   @CallSuper
   protected void onDetachedFromWindow() {
     super.onDetachedFromWindow();
-    mapZoomButtonController.setVisible(false);
+    if (mapZoomButtonController != null) {
```

Comment:
What do you think of moving the check into a `private` method?
- It increases the chances that other methods can use a method when the method is finely grained.
- It allows the higher-level methods to read more like a series of comments (`public` methods should look like pseudo code, without language statements, etc.).